metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/operations/OperationsResource.java

```diff
@@ -99,7 +99,7 @@ public Task<String> restoreIndices(@ActionParam(PARAM_ASPECT) @Optional @Nonnull
 
   @VisibleForTesting
   static boolean isTaskIdValid(String task) {
-    if (task.matches("^[a-zA-Z0-9]+:[0-9]+$")) {
+    if (task.matches("^.+:[0-9]+$")) {
```

Comment:
Does this need to be opened up fully to all characters? https://discuss.elastic.co/t/special-characters-in-elasticsearch-id-from-logstash/150968/3 Looks like it just needs to include base 64 accepted characters

Reply:
With this you could allow bad input like:

'ab:cd:ef:1'

Reply:
I see, so just `a-zA-Z0-9` along with `-` and `_`? For some reason I thought it could be any hostname?